tests/baselines/reference/autoAccessor1(target=es5).js

```diff
@@ -0,0 +1,61 @@
+//// [tests/cases/conformance/classes/propertyMemberDeclarations/autoAccessor1.ts] ////
+
+//// [autoAccessor1.ts]
+class C1 {
+    accessor a: any;
+    accessor b = 1;
+    static accessor c: any;
+    static accessor d = 2;
+}
+
+
+
+
+!!!! File autoAccessor1.js missing from original emit, but present in noCheck emit
+//// [autoAccessor1.js]
+var __classPrivateFieldGet = (this && this.__classPrivateFieldGet) || function (receiver, state, kind, f) {

```

Comment:
The `target` error for auto-accessors, `Properties with the 'accessor' modifier are only available when targeting ECMAScript 2015 and higher.` is a checker error. So when it's suppressed, we happily do emit.